bigquery/stackoverflow.rb

```diff
@@ -28,17 +26,13 @@
 # [END bigquery_simple_app_client]
 
 # [START bigquery_simple_app_query]
-sql = <<-END_SQL
-SELECT
-  CONCAT(
-    'https://stackoverflow.com/questions/',
-    CAST(id as STRING)) as url,
-  view_count
-FROM `bigquery-public-data.stackoverflow.posts_questions`
-WHERE tags like '%google-bigquery%'
-ORDER BY view_count DESC
-LIMIT 10
-END_SQL
+sql = "SELECT " +
+      "CONCAT('https://stackoverflow.com/questions/', " +
+      "       CAST(id as STRING)) as url, " +
+      "view_count " +
```

Comment:
Question: `view_count` should be able to fit on the previous line. Is there a reason to keep it on a separate line?

Reply:
not really.